bin/main.dart

```diff
@@ -58,58 +87,142 @@ _translateLocalizedFiles(List<String> langCodes, String dirPath) async {
     exit(0);
   }
 
-  Map<String, Map<String, String>> oldLangStrMap = Map();
-  Map<String, Map<String, String>> newLangStrMap = Map();
-
+  // loading keys and strings to langStringMap for each language
+  Map<String, Map<String, String>> langStringMap = Map();
```

Comment:
```suggestion
  final Map<String, Map<String, String>> langStringMap = Map();
```

Reply:
If you will not change this statement, let's keep it final, because during Dart linting it can throw a warning about this. 
Btw, the adding new elements to the collection does not count and they can be final :) But if you set a new value to the whole collection, then the collection can not be final. In this current case, as I see in the code, it should be final

Reply:
Thanks for the detailed explanation. Fixed.